extensions/2.0/Vendor/MSFT_RigidBodies/README.md

```diff
@@ -0,0 +1,159 @@
+However if a descendant node has its own `rigidBody` properties, that should be treated as an independent rigid body during simulation - there is no implicit requirement that it follows its 'parent' rigid body.
+
+If a rigid body node's transform is animated by animations in the file, those animations should take priority over the physics simulation. Rigid bodies should follow the transforms provided by the animations.
+
+Rigid bodies have the following properties:
+
+| |Type|Description|
+|-|-|-|
+|**isKinematic**|`boolean`|Treat the rigid body as having infinite mass. Its velocity will be constant during simulation.|
+|**mass**|`number`|The mass of the rigid body. Larger values imply the rigid body is harder to move.|
+|**inertiaTensor**|`number[9]`|Mass distribution of the rigid body in local space. Larger values imply the rigid body is harder to rotate.|
+|**centerOfMass**|`number[3]`|Center of mass of the rigid body in local space.|
+|**linearVelocity**|`number[3]`|Initial linear velocity of the rigid body in local space.|
+|**angularVelocity**|`number[3]`|Initial angular velocity of the rigid body in local space.|
+
+### Colliders
+
+To specify the geometry used to detect overlaps, we use the MSFT\_CollisionPrimitives extension. By default, a node with a collision primitive should not participate in the physics simulation, as that geometry may be used for application-specific use-cases. In order to indicate that a `collider` should be used in the physics simulation, that node should have a `physicsMaterial` property, which indicates that it generates some physical response from interactions with other colliders. A collider attached to a node with `MSFT_RigidBodies` should be interpreted as an object which cannot generate a collision response (i.e. no impulses are applied as a result of collision detection), but instead, should be used as a "trigger", which the application may utilize to generate callbacks for implementation of application-specific logic.
+If the node is part of a rigid body (i.e. itself or an ascendent has `rigidBody` properties) then the collider belongs to that rigid body and should move with it during simulation.
+Otherwise the collider exists as a static object in the physics simulation which can be collided with but can not be moved.
+
+Implementations of this extension should ensure that collider transforms are always kept in sync with node transforms - for example animated node transforms should be applied to the physics engine (even for static colliders).
+
+Note that, depending on the simulation engine in use, `convex` and `trimesh` colliders may impose a large computational cost when converting to native types if the source mesh contains many vertices. In addition, real-time engines generally prefer to avoid collisions between two `trimesh` objects. For best performance and behaviour, consult the manual for the physics simulation engine you are using.
```

Comment:
```suggestion
Note that, depending on the simulation engine in use, `convex` and `trimesh` colliders may impose a large computational cost when converting to native types if the source mesh contains many vertices. In addition, real-time engines generally prefer to avoid collisions between two `trimesh` objects. For best performance and behavior, consult the manual for the physics simulation engine you are using.
```

Reply:
Thanks for finding all those, Will! I really should have enabled spellcheck (and set my language to en_US :))